src/main/java/filter/expression/Qualifier.java

```diff
@@ -398,6 +399,15 @@ private static boolean isMetaQualifier(Qualifier q) {
             case "date":
                 comparator = (a, b) -> a.getUpdatedAt().compareTo(b.getUpdatedAt());
                 break;
+            case "nonselfupdate":
+                // Purposefully inverted a and b to make latest issues show up first.
+                if (metadataRefresh) {
+                    comparator = (a, b) ->
+                        b.getMetadata().getNonSelfUpdatedAt().compareTo(a.getMetadata().getNonSelfUpdatedAt());
+                } else {
+                    comparator = (a, b) -> b.getUpdatedAt().compareTo(a.getUpdatedAt());
+                }
+                break;
```

Comment:
We can compose sort comparators currently, right? The current case is that if there is an explicitly specified sort comparator, it will override `nonSelfUpdate`, which might not be what we want, in which case the user should be given the control to maintain the `nonSelfUpdate` order alongside other sort comparators, too. For example, sorting by repo, and then by `nonSelfUpdate`.

But if we allow the `nonSelfUpdate` order to be lost when another sort order is explicitly specified then extracting it to another method without touching `getCompoundSortComparator` does make sense.